src/psyclone/tests/test_files/gocean1p0/new_iteration_space.f90

```diff
@@ -0,0 +1,53 @@
+!-------------------------------------------------------------------------------
+! (c) The copyright relating to this work is owned jointly by the Crown,
```

Comment:
Please correct this copyright/license notice to be the current version.

Reply:
Done